CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+Contributing to DAGMC
+=====================
+
+The following is a guide to contributing to the DAGMC-Viz repository.
```

Comment:
"DAGMC-Viz" -> "DAGMC"  (maybe a quick search of all files for this?)

Reply:
Whoops. Fixed that.